packages/create-sitecore-jss/src/templates/nextjs/scripts/generate-config.ts

```diff
@@ -1,4 +1,3 @@
-import 'dotenv/config';
```

Comment:
I believe we also need to remove the dotenv/config import from `edge-platform.ts`